### PR TITLE
Added info from support article about flex

### PR DESCRIPTION
**Pull request by rkoron007**
Background link: https://hashicorp.atlassian.net/browse/IPE-854

### CODE REVIEW

File: website/docs/cloud-docs/overview/activate-flex.mdx [hunk 1]
```diff
@@ -44,10 +44,20 @@ Organization members may be owners or admins in only the HCP Terraform or HCP or
...
 - The HCP admin manages the billing relationship with HashiCorp Cloud Platform and owns all the HCP Terraform organizations. They can use either approach and invite HCP admins to HCP Terraform organizations or make HCP Terraform organization owners into HCP admins.
 
-## Step 3: Verify HashiCorp Flex balance allocation
+## Step 3: Activate your Flex plan
 
-Navigate to the **HCP Org > Billing**.  If HashiCorp sales has allocated Flex, the page displays a **Contract Summary** with an available balance. If it only shows your credit card information or a $50 free trial credit, your HashiCorp Flex account may be scheduled to be allocated at a future date. Please contact your **HashiCorp Sales Account Manager** for more information.
+Your sales account manager will email your organization's tech contact a code to activate your HCP Flex plan a day or two before your plan begins. 
 
+You can self-activate your Flex plan by completing the following steps:
```
Comment (trujillo-adam) [inline website/docs/cloud-docs/overview/activate-flex.mdx:51]:
```suggestion
Complete the following steps to activate your Flex plan:
```
Simpler?

File: website/docs/cloud-docs/overview/activate-flex.mdx [hunk 2]
```diff
@@ -44,10 +44,20 @@ Organization members may be owners or admins in only the HCP Terraform or HCP or
...
+1. Copy the HCP Flex activation code from your email.
+1. Click the link in your activation email to access your HCP organization or log in to your HCP portal. Then, navigate to **Billing > Payment Methods** and find the **Contract** section.
```
Comment (trujillo-adam) [inline website/docs/cloud-docs/overview/activate-flex.mdx:53]:
```suggestion
1. Click the link in your activation email to access your HCP organization or log into your HCP portal. 
1. Navigate to **Billing > Payment Methods** and find the **Contract** section.
```

**Review by trujillo-adam — APPROVED** [review]
Looks good - a few super minor nits that you can ignore if you don't agree.

File: website/docs/cloud-docs/overview/activate-flex.mdx [hunk 3]
```diff
@@ -14,7 +14,7 @@ Follow these steps to activate HashiCorp Flex for a Standard or Plus Edition HCP
 
 There are several prerequisites for activating HashiCorp Flex:
 
-1. You must email the HCP organization ID with the HashiCorp sales team.
```
Comment (leizerbeam) [inline website/docs/cloud-docs/overview/activate-flex.mdx:17]: Hi @rkoron007 this pre req can be substituted with. “Create an HCP org.” Instructions. We no longer need to know the org id they will be using.

**Pull request merged**